opm/parser/eclipse/EclipseState/SimulationConfig/SimulationConfig.hpp

```diff
@@ -31,6 +31,15 @@ namespace Opm {
 
     public:
 
+        enum Kr_model {
```

Comment:
It'll be extended with relperm options (from the SATOPTS keyword), and that's a part of the change.